dashboard/app/dsl/evaluation_question_dsl.rb

```diff
@@ -0,0 +1,10 @@
+class EvaluationQuestionDSL < MultiDSL
+  def answer(text, weight = 1, learning_module = nil)
+    answer = {text: text}
+    answer[:weight] = weight || 1
+    raise 'Weight must be an integer' unless answer[:weight].is_a?(Integer)
```

Comment:
nit: should we raise before assigning?  i know it doesn't matter much, but it feels more proper to check first.